MANIFEST.in

```diff
@@ -52,6 +52,8 @@ include docs/Dockerfile
 include ui/Dockerfile
 include ui/cap-react/Dockerfile
 include ui/cap-react/LICENSE
+recursive-include scripts *.j2
```

Comment:
`.j2` is not needed. you can just name your template with `.yml` extension. it is the same thing